addins/CFSample/customfunctions.js

```diff
@@ -303,6 +404,8 @@ CustomFunctions.associate('STOCK_PRICE_HISTORY', stockPriceHistory);
 CustomFunctions.associate('DISPLAY_WEB_DIALOG', displayWebDialog);
 CustomFunctions.associate('BITCOIN_STREAM', bitcoinStream);
 CustomFunctions.associate('ERROROUT', errorOut);
+CustomFunctions.associate('CUSTOM_ERROR_RETURN customErrorReturn', customErrorReturn);
```

Comment:
Pick one spelling. It looks like 'CUSTOM_ERROR_RETURN' is in line with the existing function IDs.

Reply:
spelling mistake. Will change to 'CUSTOM_ERROR_RETURN

Reply:
You still have 2 words in the ID.